### ISSUE TITLE
Accessibility | Focus problem with rows in the translation table

### ISSUE THREAD

**Issue by f4-lex** [Frachtwerk/essencium-frontend#837]
**Describe the bug**
You cannot open and close subrows in the translation table using keyboard navigation and thus are unable to edit translations.

- [ ] update `MIGRATION.md`
- [ ] update docs
